dist/htmlminifier.js

```diff
@@ -1162,6 +1162,14 @@
     return text;
   }
 
+  function uniqueId(value) {
```

Comment:
1. replaces `'htmlmin' + (Math.random() + '').slice(2)`, ensure that the identifier generated would not collide with input
  - the loop will almost never be executed more than once
2. replaces `<!-- htmlmin:temp -->` to avoid potential collision with input